tree/dataframe/src/RDataFrame.cxx

```diff
@@ -1780,6 +1795,9 @@ more of its entries. For example:
 - When joining different datasets horizontally according to some index value
   (e.g. the event number), if the index does not find a match in one or more
   other datasets for a certain entry.
+- If, for a certain event, the value of a certain column is invalid because
+  it results from a previous processing which involved systematic variations
+  and that value was removed by a selection. For more details, see \ref snapshot-with-variations.
```

Comment:
```suggestion
- If, for a certain event, a column is invalid because it results from a snapshot with systematic variations, and that variation didn't pass its filters. For more details, see \ref snapshot-with-variations.
```